### PR TITLE
authorization on backend

### PR DESCRIPTION
**Pull request by kevyogi**
-confirmed that deserializing works on backend
-authentication works on backend
-confirmed passwords hash
-created filter/map component to render items
-fixed some front end routes to align with proxy
-tried axios to see if that fixed proxy problem(it doesn't)
-fixed cors error

### CODE REVIEW

**Comment by kevyogi** [comment]
[Create an Auth User Home view](https://trello.com/c/GtT7LfBl/11-create-an-auth-user-home-view)

File: server/routes/Items.js [hunk 1]
```diff
@@ -30,8 +30,8 @@ router.post('/new', isAuthenticated, (req, res) => {
   });
 });
 
-router.get('/', isAuthenticated, (req, res) => {
-  console.log(req.user);
+router.get('/', (req, res) => {
+  console.log('something');
```
Comment (jesse-copeland) [inline server/routes/Items.js:34]: do not leave debugging console.log when merging to development

File: server/server.js [hunk 2]
```diff
@@ -12,13 +12,13 @@ const app = express();
...
 const PORT = process.env.PORT || 8080;
 
-app.use(function (req, res, next) {
- res.setHeader('Access-Control-Allow-Origin', 'http://localhost:3000');
- res.setHeader('Access-Control-Allow-Methods', 'GET, POST, OPTIONS, PUT, PATCH, DELETE');
- res.setHeader('Access-Control-Allow-Headers', 'X-Requested-With,content-type');
- res.setHeader('Access-Control-Allow-Credentials', true);
- next();
-});
+// app.use(function (req, res, next) {
```
Comment (jesse-copeland) [inline server/server.js:15]: remove from code before merging

File: src/actions/items.js [hunk 3]
```diff
@@ -1,17 +1,30 @@
...
 import { addNewItem } from '../lib/POSTrequests';
 import { itemDelete } from '../lib/DELETErequests';
 import { itemEdit } from '../lib/PUTrequests';
 
+const axios = require('axios');
 export const LOAD_ITEMS = 'LOAD_ITEMS';
 export const LOAD_ITEM = 'LOAD_ITEM';
 export const ADD_ITEM = 'ADD_ITEM';
 export const DELETE_ITEM = 'DELETE_ITEM';
 export const EDIT_ITEM = 'EDIT_ITEM';
```
Comment (jesse-copeland) [inline src/actions/items.js:13]: If you are no longer using code, remove it.

File: src/containers/App/App.js [hunk 4]
```diff
@@ -1,31 +1,49 @@
...
-
+import { loadItems } from '../../actions/items';
+import { loadCategories } from '../../actions/categories';
+import { loadConditions } from '../../actions/conditions';
 import ItemContainer from '../Item/itemContainer';
 import NewItemForm from '../NewItem/newItemForm';
 import Select from '../../components/select';
+import FilterMap from '../../components/FilterMap';
 
 class App extends Component {
```
Comment (jesse-copeland) [inline src/containers/App/App.js:14]: As good practice you should pass props to the constructor and to super

**Review by jesse-copeland — APPROVED** [review]
Pretty minor changes. Just try to add them in in future commits.

**Pull request merged**